tests/autoyast/verify_default_target.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Test module to verify that actual default target corresponds to the
+# expected one.
```

Comment:
I guess it fits in the previous line.

Reply:
Sorry, did not get what you are pointing to. Could you please clarify what do you want to be changed here?

Reply:
really silly thing, that I think that the Summary can be in one line, but I didn't measure characters or something like, was for the feeling how is displayed in github.

Reply:
In case we are using 80 char line length convention, it is not fit in one line. Do you know if we are using 120 or 80 line width convention? I thought it is 80.

Reply:
no idea, so if you want to keep 2 lines is totally fine for me :) but then it should apply for code as well? i.e: line 30.
At least for checking in github is nice that the lines fits in the web page but this maybe is something that I just invented not that you have to follow :) (and many times I forget to follow)